<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-U0GR5G6" ignored="0" name="Default test" tests="3" failures="0" timestamp="14 Mar 2018 15:24:31 GMT" time="0.111" errors="0">
  <testcase name="@BeforeTest Starttest" time="0.058" classname="RelearnPack.ExtentReportcls">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.io.File.<init>(Unknown Source)
at com.relevantcodes.extentreports.Report.setFilePath(Report.java:527)
at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:80)
at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:375)
at RelearnPack.ExtentReportcls.Starttest(ExtentReportcls.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest Starttest -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterTest endTest" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- @AfterTest endTest -->
  <testcase name="failTest" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- failTest -->
  <testcase name="passTest" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- passTest -->
  <testcase name="skipTest" time="0.0" classname="RelearnPack.ExtentReportcls">
    <skipped/>
  </testcase> <!-- skipTest -->
</testsuite> <!-- Default test -->
